case-lib/lib.sh

```diff
@@ -23,7 +23,8 @@ if [ ! -f "$SOF_LOCK" ];then # lock is not exist
     echo $$ > /tmp/sof-test.lock # write self pid into lock file
 elif [ ! "$(alias |grep -i 'Sub-Test')" ]; then # not the sub test-case
     echo "Find $SOF_LOCK already exist: $(ps -p $(cat $SOF_LOCK))"
-    exit 2 # now skip to run the test-case
+    # exit 2 # now skip to run the test-case
+    exit 1 # mark test-case is failed
```

Comment:
```suggestion
    exit 1 # mark test-case as failed
```
Additionally, you've got typos in your commit message -: exsit -> exist; as -> is